reservation/views.py

```diff
@@ -65,21 +69,62 @@ def select_seats(request, movie, schedule):
                 details=schedule_object,
                 seat=seat_obj,
             )
+
             reservations.append(user_reservation)
 
-        send_tickets(request, reservations, schedule_object)
-        return render(request, 'reservation/success.html')
+        confirm_reservation(request, reservations, schedule)
+        return render(request, 'reservation/reservation_email_sent.html')
 
     return render(request, 'reservation/select_seats.html', context)
 
 
-def send_tickets(request, reservation_ids, schedule):
+def success(request, group):
+
+    reservations = list(map(int, group.split('-')))
+
+    for reservation in reservations:
+        Reservation.objects.filter(
+            id=reservation
+        ).update(confirmed=True)
+
+    send_tickets(request, reservations)
+
+    return render(request, 'reservation/success.html')
+
+
+def send_tickets(request, reservations):
     subject, from_email, to = 'Your tickets', 'office@silverscreen.com', (request.user.email,)
     email_text = get_template('reservation/reservation_successful.html')
+
+    tickets = [Reservation.objects.get(id=reservation) for reservation in reservations]
+
+    context = {
+        'user': request.user,
+        'tickets': tickets,
+    }
+
+    html_content = email_text.render(context)
+    msg = EmailMessage(
+        subject=subject,
+        body=html_content,
+        from_email=from_email,
+        to=to
+    )
+    msg.send()
+
+
+def confirm_reservation(request, reservations, schedule):
+    subject, from_email, to = 'Confirm your reservation', 'office@silverscreen.com', (request.user.email,)
+    email_text = get_template('reservation/confirm_reservation.html')
+
+    user_reservations = [str(reservation.id) for reservation in reservations]
+    group = '-'.join(user_reservations)
```

Comment:
No changes are needed here, but I just wanted to let you know that you could still use values_list here, if reservations is a query set.